### ISSUE TITLE
Rams with 4 chips

### ISSUE THREAD

**Issue by asturur** [srg320/Saturn_MiSTer#1]
hi @srg320 , i read the readme, where is specified that the first ram should have 2 chips.
I assume you are doing something special with the 2 chip configuration, and i have a setup with 128mega on 4 chips, and i wonder if that can be an issue.

Do you know anything about that?

thanks for the fantastic work!

**Comment by srg320** [comment]
Is this an official module? Is there a schematic of the module?

**Comment by asturur** [comment]
i think i can dig the schematic, no is not official, there is only a single
official module per ram size, and i m not asking for support of it, just
your opinion if it can work normally. Otherwise i have to organize myself
to find an alternative

Il post the schematic if i find them!

**Comment by srg320** [comment]
It is important how the CS signals of all the chips are connected, I can only find this out from the schematic of the module.

**Comment by asturur** [comment]
Ok schematics are not publicly available.
I have been told this:
```
That's I am using pair of 32M8 chips instead single 32M16 chip. The rest
is the same
```
And if you are interested to know if this is a workable solution ( all antonio villena designs are based on this ) i can buy you a module to test.

Again, i don't pretend or think this module to be supported, i have 2 misters, and if this module is not compatible i can just use the other one.

I m asking because i m getting subpar results on testing, i can NEVER get out of the bios and load anything, all i managed to see is a couple seconds of `cotton 2`.
Eventually if the 4 chip designs is either not good, or can offer other kinds of optimizations.

So if you are curious i can send you a module.
This is the module:
https://www.antoniovillena.es/store/product/sdram-module/

**Comment by sorgelig** [comment]
There is official design of SDRAM modules. Non-standard modules should be made as 100% compatible in order to work. If not, don't ask it here. Go do developer of that module and ask him for compatibility. Don't bother developer to support non-standard HW!

**Comment by asturur** [comment]
Hey @srg320 i wasn't aware i was bothering you. I thought it was fine to ask questions out of curiosity.
I m also sorry probably my english wasn't clearly enough when i tried to clearly state that i wasn't expecting any support, and if you were curious i could send you a module, of course without expecting anything in return.

**Comment by asturur** [comment]
@sorgelig i think you either can't read very well or you didn't bother reading.
From the conversation it was clear that:

- we know there are official SDRAM modules
- It was already asked to the developer for compatibility
- no one asked for support of any kind

Also if it was the mister-devel org i perfectly knows the code of conduct and the stance and i wouldn't have asked, since this is a different repo, from a different person, with not specified code of conduct, until not clearly stated, the `don't ask it here` is probably just in your head.

You are free to use github as you want but don't pretend that every repository or code sharing works as you intend.

if @srg320 didn't feel the urge to shut me off as soon as i asked a question, maybe is just because he is, contrary to you, a nice person open to converstaion and can decide by himself when is time to cut the conversation.

So thank you for your unsolicited opinion anyway.

**Comment by sorgelig** [comment]
Yeah, i didn't notice it's not in MiSTer-devel. It pops up in my notification list.
Probably i have to remove myself from notifications of this repository.

**Comment by srg320** [comment]
@asturur, you can't send me anything because I live under occupation. I recommend waiting for the official release. If it won't work with the official release, you will need to purchase an official SDRAM module 128Mb.

**Comment by asturur** [comment]
I have plenty already of official ram modules ( 4 of them ). As i tried to make clear, this was just curiosity.

**Comment by asturur** [comment]
Just an heads up that since commits from september 15 the 4 chip ram started to boot games and while it was still unstable that is great progress.

I see there are more commits around that matter and i ll test them as soon as i have time.
